packages/console/src/onboarding/pages/CreateTenant/index.tsx

```diff
@@ -81,14 +81,13 @@ function CreateTenant() {
       if (collaboratorEmails.length > 0) {
         // Should not block the onboarding flow if the invitation fails.
         try {
-          await Promise.all(
-            collaboratorEmails.map(async (email) =>
-              tenantCloudApi.post('/api/tenants/:tenantId/invitations', {
-                params: { tenantId: newTenant.id },
-                body: { invitee: email.value, roleName: TenantRole.Collaborator },
-              })
-            )
-          );
+          await tenantCloudApi.post('/api/tenants/:tenantId/invitations', {
+            params: { tenantId: newTenant.id },
+            body: {
+              invitee: collaboratorEmails.map(({ value }) => value),
```

Comment:
nit: should be `invitees` instead of singular form `invitee`

Reply:
Change on the input param's name could bring breaking change. Maybe a good way is to first allow string array typed input `invitees` in the `POST /invitations` API.